book/src/developers/contributing/releases/deployment.md

```diff
@@ -1,37 +1,100 @@
-# Deployment
+# Deploy trin to network
 
-## Update testnet node images
-Run `make create-docker-image` and `make push-docker-image` commands with the appropriate version.
-
-Example:
-
-```sh
-make create-docker-image version=0.2.0-alpha.3
-make push-docker-image version=0.2.0-alpha.3
+## First time Setup
+- Get access to cluster repo (add person to @trin-deployments)
+- Download cluster repo:
+```shell=
+git clone git@github.com:ethereum/cluster.git
+cd cluster
+python3 -m venv venv
+. venv/bin/activate
+pip install ansible
 ```
+- Publish your pgp public key with keybase, using: `keybase pgp select --import`
+  - This fails if you don't have a pgp key yet. If so, create one with `gpg --generate-key`
+- [Install sops](https://github.com/ethereum/cluster/tree/master/portal-network/trin/ansible)
+- Contact `@paulj`, get public gpg key into cluster repo
+- Make sure your pgp key is working by running:
+  ```sops portal-network/trin/ansible/inventories/dev/group_vars/secrets.sops.yml```
+- Log in to docker with: `docker login`
+- Ask Nick to be added as collaborator on Docker repo
 
-## Deploy testnet nodes
+## Each Deployment
 
-Run the Ansible playbook to fetch the newly available docker image and update the testnet nodes.
+### Prepare
+- Announce in Discord #trin that you're about to run the deployment
+- Make sure to schedule plenty of time to react to deployment issues
 
-## Communicate
+### Update docker images
+Docker images are how Ansible moves the binaries to the nodes. Update the docker tags with:
+```shell=
+docker pull portalnetwork/trin:latest
+docker pull portalnetwork/trin:latest-bridge
+docker image tag portalnetwork/trin:latest portalnetwork/trin:testnet
+docker image tag portalnetwork/trin:latest-bridge portalnetwork/trin:bridge
+docker push portalnetwork/trin:testnet
+docker push portalnetwork/trin:bridge
+```
 
-Notify in Discord chat about the new release being complete, and the network nodes being updated.
+### Run ansible
+- Check monitoring tools to understand network health, and compare against post-deployment, eg~
+    - [Glados](http://glados.ethportal.net/content/)
```

Comment:
I think adding the `trin-bench` site here (https://trin-bench.ethdevops.io/login)  to evaluate pre/post deploy network health is also valuable.

Reply:
👍🏻 I'll put a link straight to the trin metrics dashboard. Some follow-up work would be cool to write up some hints about what to look for in Grafana, and link to that here.